app/views/posts/show.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1><%= @post.title %></h1>
-<p><%= @post.content %></p>
+<div class="post">
+<ul class="comments2">
+  <h1><%= @post.title %> by <span class="username"><%= @post.author.name %></span></h1>
+  <p class="post-details">Comments: <%= @post.comments.count %>, Likes: <%= @post.likes_counter %></p>
+  <p class="post-content"><%= @post.text %></p>
+</ul>
+  
+  <ul class="comments2">
+  <h2>Comments</h2>
+    <% @post.comments.each do |comment| %>
+      <li>
+        <strong><%= comment.user.name %>:</strong>
+        <p><%= comment.text %></p>
+      </li>
+    <% end %>
+  </ul>
+  <%= link_to 'Back', request.referer || root_path %>
+</div>
```

Comment:
- [ ] Kindly ensure the top part of the Post show page should follow the given wireframe.

Given design
![image](https://github.com/jlberbesi/blog-app/assets/81932478/bdc4458f-c18c-4c63-bed2-cdb97fe074d3)

Your design
![image](https://github.com/jlberbesi/blog-app/assets/81932478/1bc86d38-3bce-402a-965d-45f79eed7549)

- Ensure the comments and likes are aligned to the right as is in the wireframe

Reply:
![Imagen de WhatsApp 2023-12-02 a las 13 36 30_81445d16](https://github.com/jlberbesi/blog-app/assets/88105477/6c574462-b575-4593-8f39-a27d2818392f)